app/controllers/keywords_controller.rb

```diff
@@ -1,13 +1,20 @@
 # frozen_string_literal: true
 
 class KeywordsController < ApplicationController
-  def index; end
+  def index
+    pagy, keywords = pagy(KeywordsQuery.new.call)
+
+    render(locals: {
+             pagy: pagy,
+             presenters: keywords
```

Comment:
Avoid plurals and be more descriptive (as there could be more than one presenter):

```suggestion
             keyword_presenter: keyword_presenter
```

Reply:
The presenter is implemented in the frontend PR. So I'll just remove this line for now.

Fixed in d36d4221b027746402452deaea288f98810af599